06-find.md

```diff
@@ -492,3 +492,24 @@ about them."
 > of the four sisters is mentioned?  Hint: one solution might employ
 > the commands `grep` and `wc` and a `|`, while another might utilize
 > `grep` options.
+
+> ## Planet names {.challenge}
+>
+> The file `planets.txt` inside the folder `nelle/data` contains a list
+> of planets and astronomical information about them. Each line is
+> a different planet, and the first is the header.
+> Find the command to count the number of planets that match a given
```

Comment:
I think that we could improve this sentence so it will be more clear for novices. Maybe replace
"Find the command to count the number of planets that match a given word." with "Find the command that receive a word as input and print on the screen the number of planets in `planets.txt` whose name has that word as part of the name."?